docs/source/examples/elasticsearch_outlet.rst

```diff
@@ -0,0 +1,85 @@
+ElasticSearch Outlet
+--------------------
+
+.. container:: tutorial-block
+
+    In this example we create an implementation of :any:`Outlet` that indexes documents to a running Elasticsearch instance. 
+    
+    **Note**: this example assumes that Elasticsearch is correctly configured and that the index you are indexing documents to exists
+    with the appropriate mappings.  
```

Comment:
Cool! Maybe we could drop a link to Elasticsearch docs here to guide users who'd like try this Outlet but haven't set ES up yet?

Reply:
Cool, I'm going to link to the official Elasticsearch python client itself rather than Elasticsearch's generic ones as its the client that is being used.